launchcontainers/launchcontainers.py

```diff
@@ -20,6 +20,8 @@ def generate_cmd(new_lc_config,sub,ses,Dir_analysis, path_to_analysis_config_jso
     host = new_lc_config['general']['host']
     sif_path=  new_lc_config['general']['sif_path']  
     
+    containersif = os.path.join(sif_path,container+'_'+new_lc_config['container_specific'][container]['version']+'.sif')
```

Comment:
This edit is in debate, Gari insists on reading from the analysis folder instead of reading from command line input when we are in the --run_lc mode. I am still considering this thing, the potential solution is to add a option in the yaml file, so that the user can choose which config they want to use.

Reply:
I am kinda thinking to go back to the way you are using. This change is too specific, we need to have a global variable that passed to all the pipeline cmd, not only just DWI pipeline